src/Panel/panel.scss

```diff
@@ -92,6 +92,14 @@ $content-title-color: $brc-darkGrey !default;
     border-left: 1px solid $border-color;
   }
 
+  &.floating {
```

Comment:
> do we have some collapsable-left panel with the rotated title visible when collapsed

yes, in AMS. If it's not a problem I would revert it to avoid a breaking change. As it's a small fix we can do it later on once we've finished the re-styling